tests/test_test_utils.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import mock
+from urlparse import urlparse
+from nose.tools import *  # flake8: noqa
+import unittest
+
+from framework.auth import Auth
+
+from website.models import Node, NodeLog
+
+from tests.base import OsfTestCase
+from tests.factories import ProjectFactory
+from tests import utils as test_utils
+
+class TestUtilsTests(OsfTestCase):
+
+    def setUp(self):
+        super(TestUtilsTests, self).setUp()
+
+        self.node = ProjectFactory()
+        self.user = self.node.creator
+        self.auth = Auth(self.user)
+
+    def test_assert_logs(self):
+        
+        def add_log(self):
+            self.node.add_log(NodeLog.UPDATED_FIELDS, {}, auth=self.auth)
+        wrapped = test_utils.assert_logs(NodeLog.UPDATED_FIELDS, 'node')(add_log)
+        wrapped(self)
+
+    def test_assert_logs_fail(self):
+        
+        def dont_add_log(self):
+            pass
+        wrapped = test_utils.assert_logs(NodeLog.UPDATED_FIELDS, 'node')(dont_add_log)
+        assert_raises(AssertionError, lambda: wrapped(self))
```

Comment:
Minor, but you can use assert_raises as a context manager, which is slightly more readable:

```python
with assert_raises(AssertionError):
    wrapped(self)
```